src/gpuarray_reduction.c

```diff
@@ -0,0 +1,548 @@
+/* Includes */
+#ifdef _MSC_VER
+#define _CRT_SECURE_NO_WARNINGS
+#endif
+
+#include <assert.h>
+#include <stdarg.h>
+#include <stddef.h>
+#if !defined(_MSC_VER) || _MSC_VER < 1600
+#include <stdint.h>
+#endif
```

Comment:
`#include gpuarray/config.h` instead.  It does that dance for stdint and has a fallback for older MSVC.